examples/preprocessing/plot_scaling_importance.py

```diff
@@ -137,3 +179,83 @@
+# dimensionality results in components with the same order of magnitude. In this
+# case it also improves the separability of the clases. Indeed, in the next
+# section we confirm that a better separability has a good repercussion on the
+# overall model's performance.
+#
+# Effect of rescaling on model's performance
+# ==========================================
+#
+# First we show how the optimal regularization of a
+# class:`~sklearn.linear_model.LogisticRegressionCV` depends on the scaling or
+# non-scaling of the data:
+
+import numpy as np
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import LogisticRegressionCV
+
+Cs = np.logspace(-5, 5, 20)
+
+unscaled_clf = make_pipeline(pca, LogisticRegressionCV(Cs=Cs))
+unscaled_clf.fit(X_train, y_train)
+y_pred = unscaled_clf.predict(X_test)
+
+scaled_clf = make_pipeline(scaler, pca, LogisticRegressionCV(Cs=Cs))
+scaled_clf.fit(X_train, y_train)
+y_pred_scaled = scaled_clf.predict(X_test)
+
+print("Optimal C for the unscaled PCA")
+print(unscaled_clf[-1].C_)
+print()
+print("Optimal C for the standardized data with PCA")
+print(scaled_clf[-1].C_)
+
+# %%
+# The need for regularization is higher (lower values of `C`) for the data
+# that was not scaled before applying PCA. From the plot we can confirm that
+# less separable clases are more propense to overfitting.
+#
+# We now evaluate the effect of scaling on the accuracy:
+
+from sklearn.metrics import accuracy_score
+
+print("Test accuracy for the unscaled PCA")
+print(f"{accuracy_score(y_test, y_pred):.2%}")
+print()
+print("Test accuracy for the standardized data with PCA")
+print(f"{accuracy_score(y_test, y_pred_scaled):.2%}")
+
+# %%
+# A clear difference in prediction accuracies is observed when the data is
+# scaled before PCA, as it vastly outperforms the unscaled version. This
+# corresponds to the intuition obtained from the plot in the previous section,
+# where the components become linearly separable when scaling before using PCA.
+#
+# Notice that in this case the models with scaled features perform better than
+# the models with non-scaled features because all the variables are expected to
+# be predictive and we rather avoid some of them being comparatively ignored.
+#
+# If the variables in lower scales were not predictive, one may experience a
+# decrease of the performance after scaling the features: noisy features would
+# contribute more to the prediction after scaling and therefore scaling would
+# increase overfitting.
+#
+# Last but not least, we evaluate the effect of scaling on the mean log-loss:
+
+from sklearn.metrics import log_loss
+
+y_proba = unscaled_clf.predict_proba(X_test)
+y_proba_scaled = scaled_clf.predict_proba(X_test)
+
+print("Log-loss for the unscaled PCA")
+print(f"{log_loss(y_test, y_proba):.3}")
+print()
+print("Log-loss for the standardized data with PCA")
+print(f"{log_loss(y_test, y_proba_scaled):.3}")
+
+# %%
+# Indeed, we achieve a lower log-loss by means of the scaling step.
```

Comment:
I would merge log-loss with the accuracy above. Just print both next to each other. Interpretation and message are the same.